File: tempe.py
op = int(input("Ingrese la opción que desee: 1.-Celsius 2.-Fahrenheit\n"))
if op == 1:
    tempe = int(input("Ingrese su temperatura: "))
    cel = (tempe * 1.8) + 32
    print("Su temperatura en Fahrenheit es de: " + str(cel))
else:
    tempe = int(input("Ingrese su temperatura: "))
    far = (tempe - 32) / 1.8
    print("Su temperatura en Celsius es de: " + str(far))

File: tempe.js
const readline = require('readline').createInterface({
  input: process.stdin,
  output: process.stdout
})
let tempe, res;

readline.question("¿Cual es tu temperatura?: ", (t) => {
  tempe = parseInt(t);
})
readline.question("Elige la opción deseada: 1.- Celsius, 2.- Fahrenheit", (num) => {
  if (parseInt(num) == 1) {
    res = (num + 32) * 1.8;
  } else {
    res = (num * 1.8) + 32 
  }
})